backend/src/scripts/utility/users/identifyInactiveUsers.js

```diff
@@ -0,0 +1,96 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const DayJS = require('dayjs');
+const Path = require('path');
+const { v5Path } = require('../../../interop');
+const FS = require('fs');
+const DBHandler = require('../../../v5/handler/db');
+
+const { logger } = require(`${v5Path}/utils/logger`);
+const { getUsersByQuery } = require(`${v5Path}/models/users`);
+const { getLastLoginDate } = require(`${v5Path}/models/loginRecords`);
+
+const DEFAULT_OUT_FILE = 'inactiveUsers.csv';
+
+const formatDate = (date) => (date ? DayJS(date).format('DD/MM/YYYY') : '');
+
+const writeResultsToFile = (results, outFile) => new Promise((resolve) => {
+	logger.logInfo(`Writing results to ${outFile}`);
+	const writeStream = FS.createWriteStream(outFile);
+	writeStream.write('Username,First Name,Last Name,Email,Company,Last Login,Number of Teamspaces\n');
+	results.forEach(({ user, firstName, lastName, email, company, lastLogin, teamspaceNumber }) => {
+		writeStream.write(`${user},${firstName},${lastName},${email},${company},${formatDate(lastLogin)},${teamspaceNumber}\n`);
+	});
+
+	writeStream.end(resolve);
+});
+
+const getFileEntry = async ({ user, roles, customData }) => {
+	const lastLogin = await getLastLoginDate(user);
+	const { firstName, lastName, email, billing: { billingInfo: { company } } } = customData;
```

Comment:
getting error on staging:
![image](https://github.com/user-attachments/assets/9b8fb268-851b-4aba-8b0f-c9c0d11a14d9)


This is due to us having some system users for monitoring:
![image](https://github.com/user-attachments/assets/d92a52a4-bc98-4666-b5d9-0dcc8b7cca28)


can we make sure we prune out users that aren't proper users? actual users should have this role:
![image](https://github.com/user-attachments/assets/946f47cc-06b2-4be5-8b69-f34fd6449563)